docs/releases/ez_platform_v2.3.md

```diff
@@ -0,0 +1,112 @@
+
+**Version number**: v2.3
+
+**Release date**: October 5, 2018
+
+**Release type**: Fast Track
+
+## Notable changes
+
+### Content scheduling
+
+!!! enterprise
+
+    You can now schedule content on a Page to become visible at a specific time in the future.
+
+    To do this you can use the Schedule tab in any block's configuration or a special Content Scheduler block.
+
+    In the Schedule tab you can define when any block becomes visible and when it disappears from a Page.
+
+    ![Schedule tab](img/2.3_schedule_tab.png)
+
+    Content Scheduler is a special block with a queue of Content items, each with its own airtime.
+    The Content becomes available at the airtime, and is replaced with new Content items coming in from the queue.
+
+    ![Content Scheduler](img/2.3_content_scheduler.png)
+
+    All changes to scheduled content on a Page are visible in the timeline.
+
+    ![Timeline and list of upcoming events](img/2.3_timeline_list.png)
+
+    The timeline also shows other events, such a Content published using the date-based publisher.
+
+### Form Builder
+
+!!! enterprise
+
+    The new Form Builder enables you to create Form Content items with multiple form fields.
+
+    ![Form Builder](img/2.3_form_builder.png)
+
+    You can preview and download submissions in the Back Office.
+
+    ![Form Builder submissions](img/2.3_form_builder_submissions.png)
+
+    See [Extending Form Builder](../guide/extending_form_builder.md) for information on how to modify and create Form fields.
+
+### ImageAsset Field Type
+
+You can now create a single source media library with images that can be reused across the system.
+
+See [Reusing images](../guide/images.md#reusing-images) and [ImageAsset Field Type reference](../api/field_type_reference.md#imageasset-field-type) for more information.
+
+![Set up multiple relations with image](img/2.3_image_asset.png)
+
+### Regenerating URL aliases
+
+A new `ezplatform:urls:regenerate-aliases` command enables you to regenerate all URL aliases.
+You can use it after changing URL alias configuration, or in case of database corruption.
+
+See [Regenerating URL aliases](../guide/url_management.md#regenerating-url-aliases) for more information.
+
+### User preferences
+
+You can now access and set user preferences in the user menu.
+
+![User preferences screen with time zone settings](img/2.3_user_preferences.png)
+
+It is covered by the `user/preferences` Policy.
+
+On API side, this is handled by [`UserPreferenceService`](https://github.com/ezsystems/ezpublish-kernel/blob/master/eZ/Publish/API/Repository/UserPreferenceService.php).
+
+### Dates in preferred timezone
+
+eZ Platform can now display dates across the system using timezone from User Settings.
+
+### Improved selection in UDW
+
+Selection of content in Universal Discovery Widget has seen improvements,
+in particular when selecting multiple Content items.
+
+![Multiple selection on UDW](img/2.3_udw_selection.png)
+
+### API improvements
```

Comment:
I think this is covered in https://github.com/ezsystems/developer-documentation/blob/2.2/docs/releases/ez_platform_v2.2.0.md#simplified-use-of-content-and-languages-in-api

The improvment meant for 2.3 `content->getContentType()` did not make it, so that will be in 2.4 instead.

Reply:
@andrerom, does this mean this change should also apply since 2.3? https://github.com/ezsystems/developer-documentation/pull/413

Reply:
You mean 2.2? yes the doc is valid as of 2.2, 2.3 does not afaik add anything around this.

What is [new in 2.3 in API]
(https://github.com/ezsystems/ezpublish-kernel/compare/7.2...master/) is:
- user preferences service
- asset relation type _(for now just like FIELD relation in API, but features around it is planned for the future, and it's already used in admin UI)_
- `TrashItem->trashed` timestamp for when an item was trashed _(contributed by @thiagocamposviana btw)_

Reply:
Yes, I mean 2.2 :) And thanks for the clarification, we mixed up the versions, corrected now.